packages/react/src/components/DatePickerInput/next/DatePickerInput.js

```diff
@@ -68,7 +67,7 @@ const DatePickerInput = React.forwardRef(function DatePickerInput(props, ref) {
     [`${prefix}--date-picker--nolabel`]: !labelText,
   });
 
-  const { openCalendar, iconDescription, ...other } = { ...rest };
+  const { ...other } = { ...rest };
```

Comment:
I think this can just be `rest` now? The assignment pattern looks so funny now lol. Definitely an odd one when quickly getting rid of deprecated code

Reply:
lol! my bad. made a mental todo to refactor that as ...rest in the render.